.github/workflows/build_and_push_latest_image.yml

```diff
@@ -35,4 +35,4 @@ jobs:
           cd Dockerfiles/midstream/
           podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=master
           podman login -u="$QUAY_USERNAME" -p="$QUAY_PASSWORD" quay.io
-          podman push midstream_image:latest quay.io/samvarankashyap/cvp-midstream-image
+          podman push midstream_image:$latestTag quay.io/cvpops/cvp-midstream-image:$latestTag
```

Comment:
Where is the `$latestTag` defined ?

Reply:
I missed a line there. 
I fixed it.

Reply:
IIUC the goal of the github workflow is to push latest master after merging.
So we should not overwrite any tag. So IMHO the usage of `latest` was good

Reply:
@lslebodn 
```
on:
  # Triggers the workflow on tag events events but only for the master branch
  push:
    tags:
      - '*'
  # Allows you to run this workflow manually from the Actions tab
```
according to the syntax, this workflow only gets triggered in case of creation of new tags. 
Thus only whenever a new tag is created the image is built and pushed . 
hope this answers. We will not be overwriting any tags. 
we will only create a new tag. or push to the tags which we do not do in our case.

Reply:
@samvarankashyap  that triiger is in the different file.
I can see 
```
# This is a basic workflow to help you get started with Actions
name: "Build and push to quay on every commit"
# Controls when the action will run.
on:
  # Triggers the workflow on push or pull request events but only for the master branch
  push:
    branches: [ master ]
```
in this file therefore usage of `latest` make more sense to me

Reply:
Yes. True. I think i was talking about the other file. 
I will change it ASAP